Source/Core/DolphinWX/ARCodeAddEdit.cpp

```diff
@@ -44,8 +44,8 @@ CARCodeAddEdit::CARCodeAddEdit(int _selection, const std::vector<ActionReplay::A
 	}
 	else
 	{
-		currentName = StrToWxStr(arCodes.at(selection).name);
-		tempEntries = arCodes.at(selection);
+		currentName = StrToWxStr(arCodes->at(selection).name);
+		tempEntries = arCodes->at(selection);
```

Comment:
Per the style guide, pointers should be used instead of references if the data is modified.